egs/babel_multilang/s5/run-2c-bnf.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+# -v3 is as -v2 but it just consists 10 closest fullLP langs to GRG + GRG.
+# -v2 is as 2c-bnf.sh but it consists of FullLP GRG.
+
+# Copyright 2016  Pegah Ghahremani
+# Apache 2.0
+
+#This yields approx 70 hours of data
+# this script generates bottleneck features from multilingual model
+# trained on list of languages and dump the bnf for specific language L.
+set -e           #Exit on non-zero return code from any command
+set -o pipefail  #Exit if any of the commands in the pipeline will
+                 #return non-zero return code
+. conf/common_vars.sh || exit 1;
+
+set -u           #Fail on an undefined variable
+skip_kws=true
+skip_stt=false
+bnf_train_stage=-100
+stage=1
+num_archives=20
+bnf_init_lrate=0.0017
+bnf_final_lrate=0.00017
+bnf_layer=5
+relu_dim=600
+bottleneck_dim=42
+speed_perturb=true
+multidir=exp/nnet3/multi_bnf_10_close_lang_plus_grg
+global_extractor=exp/multi/nnet3/extractor
+lang_list=(GRG LIT MONG TUR KAZ KUR PSH SWA TOK IGBO DHO)
+use_flp=true
+
+. ./utils/parse_options.sh
+
+
+L=$1
+
+if $use_flp; then
+. local/prepare_flp_langconf.sh $L
+else
+. local/prepare_llp_langconf.sh $L
+fi
+
+langconf=langconf/$L/lang.conf
+[ ! -f $langconf ] && echo 'Language configuration does not exist! Use the configurations in conf/lang/* as a startup' && exit 1;
+. $langconf || exit 1;
+
+suffix=
+if $speed_perturb; then
+  suffix=_sp
+fi
+exp_dir=exp/$L
```

Comment:
This directory structure (`{data,exp}/<lang-name>/`) is not used in other Babel recipes. Better make it consistent with other scripts.